@stellar/design-system/src/components/Typography/index.tsx

```diff
@@ -16,7 +17,7 @@ export const Heading = ({
   ...props
 }: HeadingProps): JSX.Element => (
   <HtmlTag
-    className={`Heading Heading--${size} ${props.className || ""}`}
+    className={`Heading Heading--${size} ${props.addlClasses || ""}`}
```

Comment:
Thanks for catching this bug! A better fix might be to move `{...props}` before `className`. Right now, the original `props` are overriding the `className` we're setting. What do you think? Or would it make more sense to use a custom name for additional classes?